manifests/object/notification.pp

```diff
@@ -123,7 +123,7 @@
   if !is_string($user_groups) { validate_array($user_groups) }
   if $times { validate_hash ($times )}
   if $command { validate_string ($command )}
-  if $interval { validate_integer ($interval )}
+  if $interval { validate_re($interval, '^\d+\.?\d*[d|h|m|s]?$')}
```

Comment:
Okay, I copied it from https://github.com/Icinga/puppet-icinga2/blob/master/manifests/object/service.pp#L216 without thinking,